services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/S3RetentionPolicyTest.java

```diff
@@ -53,51 +58,47 @@
   @Autowired
   private DistributionServiceConfig distributionServiceConfig;
 
-  private List<String> generateHourFilesForDay(LocalDate day, String country) {
-    List<String> files = new ArrayList<>();
-    for (var i = 0; i <= 24; i++) {
-      files.add(generateHourFilename(day, country, Integer.toString(i)));
-    }
-    return files;
-  }
+  private LocalDateTime currentTime;
 
-  private List<S3Object> s3ObjectsFromFilenames(List<String> filenames) {
-    return filenames.stream().map(S3Object::new).collect(Collectors.toUnmodifiableList());
+  @BeforeEach
+  void setup() {
+    currentTime = getCurrentUtcHour();
   }
 
   @Test
-  void shouldDeleteHourFile() {
-    var toBeKept = generateHourFilesForDay(getUtcDate().minusDays(1), "DE");
-    var toBeDeleted = generateHourFilesForDay(getUtcDate().minusDays(2), "DE");
+  void shouldDeleteDiagnosisKeyHourFiles() {
+    var toBeKept = generateDiagnosisKeyHourFilesForDay(getUtcDate().minusDays(1), "DE");
+    var toBeDeleted = generateDiagnosisKeyHourFilesForDay(getUtcDate().minusDays(2), "DE");
 
     List<S3Object> mockResponse = this.s3ObjectsFromFilenames(Stream.concat(toBeDeleted.stream(), toBeKept.stream())
         .collect(Collectors.toUnmodifiableList()));
 
-    when(objectStoreAccess.getObjectsWithPrefix(eq(this.getPrefix("DE")))).thenReturn(mockResponse);
-    s3RetentionPolicy.applyHourFileRetentionPolicy(2);
+    when(objectStoreAccess.getObjectsWithPrefix(eq(this.getDiagnosisKeyPrefix("DE")))).thenReturn(mockResponse);
+    s3RetentionPolicy.applyDiagnosisKeyHourRetentionPolicy(2);
 
     toBeDeleted.forEach(f -> verify(objectStoreAccess, times(1)).deleteObjectsWithPrefix(eq(f)));
     toBeKept.forEach(f -> verify(objectStoreAccess, never()).deleteObjectsWithPrefix(eq(f)));
   }
 
   @Test
-  void shouldDeleteHourFileFromAllBuckets() {
-    var toBeKeptDE = generateHourFilesForDay(getUtcDate().minusDays(1), "DE");
-    var toBeKeptEUR = generateHourFilesForDay(getUtcDate().minusDays(1), "EUR");
+  void shouldDeleteDiagnosisKeyHourFilesFromAllBuckets() {
+    var toBeKeptDE = generateDiagnosisKeyHourFilesForDay(getUtcDate().minusDays(1), "DE");
+    var toBeKeptEUR = generateDiagnosisKeyHourFilesForDay(getUtcDate().minusDays(1), "EUR");
 
-    var toBeDeletedDE = generateHourFilesForDay(getUtcDate().minusDays(2), "DE");
-    var toBeDeletedEUR = generateHourFilesForDay(getUtcDate().minusDays(2), "EUR");
+    var toBeDeletedDE = generateDiagnosisKeyHourFilesForDay(getUtcDate().minusDays(2), "DE");
+    var toBeDeletedEUR = generateDiagnosisKeyHourFilesForDay(getUtcDate().minusDays(2), "EUR");
 
+    List<S3Object> mockResponseDE = this
+        .s3ObjectsFromFilenames(Stream.concat(toBeDeletedDE.stream(), toBeKeptDE.stream())
+            .collect(Collectors.toUnmodifiableList()));
+    when(objectStoreAccess.getObjectsWithPrefix(eq(this.getDiagnosisKeyPrefix("DE")))).thenReturn(mockResponseDE);
```

Comment:
Also not sure why we need the eq here? According to sonar the default matching method is already equals(). In my opinion removing the eq() calls make the code more readable. What do you think?